### PR TITLE
User story 23/public rooms

### LINKED ISSUE #23: As a player, I can join random rooms so that i can play with strangers

### PR DESCRIPTION
**Pull request by samcohen1**
A user can create and join private and public games to play with friends or other users online

 Closes #23

### CODE REVIEW

File: src/public/js/landingPage.js
Comment (ChristopherPearse) [inline src/public/js/landingPage.js:1]: The addition of public room joining and creating appears to be implemented with good coding practices.

File: src/server.js
Comment (ChristopherPearse) [inline src/server.js:1]: Is there a reason for the numerous console outputs? If this is for debugging purposes only, please comment it out or remove it before requesting to merge.

File: tests/gameRunning.spec.js
Comment (ChristopherPearse) [inline tests/gameRunning.spec.js:1]: All tests run successfully on GitHub and test conditions are valid.

**Review by ChristopherPearse — COMMENTED** [review]
Please address my concerns about your console logs. However, the rest of the code was implemented well, making logical sense for the user story at hand.

File: src/server.js
Comment (samcohen1) [reply src/server.js:1]: Will fix that, thank you

**Review by ChristopherPearse — APPROVED** [review]
Adjustments are implemented correctly according to comments. The code works, with no tests being flagged.

**Pull request merged**